ui/admin/app/templates/scopes/scope/edit.hbs

```diff
@@ -66,21 +66,110 @@
   {{/if}}
 
   <page.body>
-    {{#if (can 'read model' @model)}}
-      <Form::Scope
-        @model={{@model}}
-        @submit={{route-action 'save' @model}}
-        @cancel={{route-action 'cancel' @model}}
-      />
-    {{else}}
-      <Rose::Layout::Centered>
-        <Rose::Message as |message|>
-          <message.description>
-            {{t 'resources.scope.messages.cannot_read'}}
-          </message.description>
-        </Rose::Message>
-      </Rose::Layout::Centered>
-    {{/if}}
+    <Rose::Layout::BodyContent as |bc|>
+      <bc.Body>
+        {{#if (can 'read model' @model)}}
+          <Form::Scope
+            @model={{@model}}
+            @submit={{route-action 'save' @model}}
+            @cancel={{route-action 'cancel' @model}}
+          />
+        {{else}}
+          <Rose::Layout::Centered>
+            <Rose::Message as |message|>
+              <message.description>
+                {{t 'resources.scope.messages.cannot_read'}}
+              </message.description>
+            </Rose::Message>
+          </Rose::Layout::Centered>
+        {{/if}}
+      </bc.Body>
+      {{#if (can 'attachStoragePolicy scope' @model)}}
+        <bc.Sidebar>
+          <div class='policy-sidebar'>
+            <Hds::Text::Display @tag='h3'>
+              {{t 'resources.policy.title'}}
+            </Hds::Text::Display>
+
+            <Hds::Text::Body @tag='p' class='description'>
+              {{t 'resources.policy.titles.description'}}
```

Comment:
Is this the right translation? Shouldn't it depend on if there's a storage policy and if there isn't, it should say something about enabling?

<img width="422" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/5783847/2e8c8661-920e-48cd-bffa-ff6e797a6c20">

This doesn't make much sense otherwise if there's no policy attached.

Reply:
oops, there are two states, updated. ty!